src/book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_reader :rentals
```

Comment:
- [Optional] Hi @Diegogagan2587, I think the rentals variable can equally be modify in this class, hence let's kindly use Getters and Setters.